### DISCUSSION TITLE
Export Circuit Data to a File

### DISCUSSION THREAD

**Discussion by TripleRD** [adgt/pyQuirk#9] [Q&A]
How can we export the circuit which is created using Quirk in file format?

**Comment by amirebrahimi** [comment]
Do you mean exporting Quirk to json format?

**Reply by TripleRD** [reply]
Yes

**Reply by amirebrahimi** [reply]
You can click on the Export button:
![image](https://user-images.githubusercontent.com/904110/118296781-33126300-b492-11eb-8cbb-d394468283f5.png)

and then copy/paste from here:
![image](https://user-images.githubusercontent.com/904110/118296846-445b6f80-b492-11eb-988f-50fcd5c17452.png)

If you are looking for an API on the Python side, then we don't currently have that in pyQuirk.

**Reply by TripleRD** [reply]
Okay, thanks!
